coderadar-core/src/main/java/io/reflectoring/coderadar/query/port/driven/GetCommitsInProjectPort.java

```diff
@@ -11,4 +11,6 @@
   List<Commit> getSortedByTimestampAsc(Long projectId);
 
   List<Commit> getSortedByTimestampAscWithNoParents(Long projectId);
```

Comment:
This method is never used. Should we keep it though?

Reply:
I think we should keep it as it may be useful in the future.